conf/config_template.yaml

```diff
@@ -30,6 +32,8 @@ global:
   debug_mode: True
   coordconv_convert: False
   coordvonc_scale:
+  max_used_ram: 2000
+  max_used_perc: 15
 
 
 # Data analysis parameters; used in data_analysis.py ------------------
```

Comment:
seeing that you're modifying config_template.yaml, do you want to address issue #189 in this PR?

Reply:
and how about address #177, i.e. add a "gdl_git_hash" parameter to "save" git version for repeatability ?

Reply:
yes, propositions made in the latest commits